include/ck/tensor_operation/gpu/device/device_gemm_reduce_xdl_cshuffle.hpp

```diff
@@ -30,6 +30,7 @@ template <typename ALayout,
           typename D0ReduceOperation,
           typename D1ReduceOperation,
           typename D1ElementwiseOperation,
+          InMemoryDataOperationEnum DGlobalMemoryDataOperation,
```

Comment:
Please change to
typename DxsElementwiseOperation,
typename DxsReductionOperation,
typename DxsGlobalMemoryDataOperation,

each of them is a tuple

Reply:
[2fae013](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/209/commits/2fae013fd2782160941261f303a27f59cb5bb310)